db/migrate/20160911210531_change_courses_timeline_end_to_datetime.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeCoursesTimelineEndToDatetime < ActiveRecord::Migration
+  def up
+    change_column :courses, :timeline_end, :datetime
+    # FIXME: doesn't appear to work on the first run?
+    Course.all.each do |course|
```

Comment:
Likewise, let's do this in a separate migration, which hopefully will also fix the bug.